### ISSUE TITLE
In Livewire customContainer not taking effect

### ISSUE THREAD

**Issue by abakano21** [victorybiz/laravel-tel-input#16]
Hi everyone,
Composer.json:
"laravel/framework": "^8.83.16",
"livewire/livewire": "^2.5",
"victorybiz/laravel-tel-input": "^1.2"

Package.json: 
"intl-tel-input": "^17.0.16",

I have installed package properly, but when I tried to use customContainer, it wrapped with span tag (wire:ignore)
Please help me out, whether there is another way or am I missing some configuration
![Screenshot from 2023-01-12 13-03-44](https://user-images.githubusercontent.com/35096812/212024509-e9dfe76f-9c07-4cab-a3e7-4b4f644c4fac.png)

**Comment by MCKLtech** [comment]
I have the same issue, and is likely the cause of https://github.com/victorybiz/laravel-tel-input/issues/23

Did you happen to solve this?

**Comment by abakano21** [comment]
@MCKLtech  no, issue still persist. I used different way, dead-lines coming :)

**Comment by hsmfawaz** [comment]
The problem was in the config file for the **utilsScript** key
You should download the utilities script and put the path in the config
![image](https://github.com/victorybiz/laravel-tel-input/assets/16060516/461ef2a4-5915-4f81-b8b4-210e31db4534)

script used : https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/21.2.7/js/utils.min.js
and it will work correctly
